### PR TITLE
Doc: releasenotes_1.6.1

### PR DESCRIPTION
**Pull request by fuzhongl**
Signed-off-by: fuzhongl <fuzhong.liu@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by wenlingz** [comment]
@fuzhongl ,please refer the build log:
make[1]: Entering directory '/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/doc'

New errors/warnings found, please fix them:
==============================================

/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/release_notes/release_notes_1.6.1.rst:4: WARNING: Title underline too short.

ACRN v1.6.1 (May 2020)
####################
/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/release_notes/release_notes_1.6.1.rst:119: WARNING: Explicit markup ends without a blank line; unexpected unindent.

make[1]: Leaving directory '/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/doc'
make[1]: *** [Makefile:61: html] Error 1
make: *** [Makefile:209: doc] Error 2
Build step 'Execute shell' marked build as failure

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/release_notes/release_notes_1.6.1.rst [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+or use Git clone and checkout commands::
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v1.6.1
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v1.6.1 documents can be found at https://projectacrn.github.io/1.6.1/.
+Documentation for the latest (master) branch is found at https://projectacrn.github.io/latest/.
+ACRN v1.6.1 requires Clear Linux OS version 33050. Follow thedd
```
Comment (gvancuts) [inline doc/release_notes/release_notes_1.6.1.rst:25]:
```suggestion
ACRN v1.6.1 requires Clear Linux OS version 33050. Follow the
```

**Pull request merged**